src/components/Header/Header.tsx

```diff
@@ -44,7 +44,7 @@ const PrimaryCta = ({ children, className = '' }: { children: any; className?: s
         <li className="leading-none">
             <button
                 onClick={() => {
-                    window.location.href = 'https://app.posthog.com/signup?src=header'
+                    window.location.pathname = '/sign-up'
```

Comment:
It looks like we're hardcoding everyone to the new page?

Reply:
The point is, sending everyone through `/sign-up` works. `signupLogic` will mount with empty, broken values if you try to access it immediately, because posthog-js is not available yet. Waiting to mount `signupLogic` until a page change occurs basically guarantees that `posthogAnalyticsLogic` and posthog-js are correctly mounted.

`/sign-up` acts like a dispatcher or traffic conductor, deciding where to send the user and then sending them there. I can fix up the React rendering so the modal doesn't flash for a few milliseconds, but that's the general idea and it worked fine for my testing. (Use a few incognito sessions to be assigned random variants).

What doesn't work is trying to mount `posthogAnalyticsLogic` and `signupLogic` simultaneously, or setting up listeners across those two logics. In the next few commits, I have tried, and failed, to navigate around the peculiarities of Kea, when in vanilla Redux I know exactly how to express how this should work.